src/main/resources/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.R

```diff
@@ -97,6 +97,16 @@ ggplot(yieldData) +
   labs(x="Read Pairs", y="Count of Double-Strand Families and Duplexes", title=paste("Duplex Yield by Input Read Pairs for", sampleInfo)) +
   theme(plot.title = element_text(hjust = 0.5), legend.title=element_blank())
 
+# Plot #4b - Duplex Yield
+ggplot(yieldData) +
+  aes(x=read_pairs) +
+  geom_line(aes(y=ds_fraction_duplexes / ds_fraction_duplexes_ideal, color="Actual / Ideal Duplexes")) +
+  scale_fill_manual(values=fgcolors) +
```

Comment:
I think given that you're using `geom_line` you want `scale_fill_color` here (color is for the line, fill is for the, umm, fill).

Reply:
Ugh, typo, sorry.  `scale_color_manual()`